src/System.Diagnostics.Process/tests/ProcessTest_ConsoleApp/ProcessTest_ConsoleApp.cs

```diff
@@ -7,45 +8,44 @@ class Program
     {
         static int Main(string[] args)
         {
-            try
+            if (args.Length > 0)
             {
-                if (args.Length > 0)
+                if (args[0].Equals("infinite"))
                 {
-                    if (args[0].Equals("infinite"))
-                    {
-                        // To avoid potential issues with orphaned processes (say, the test exits before
-                        // exiting the process), we'll say "infinite" is actually 30 seconds.
-                        
-                        Console.WriteLine("ProcessTest_ConsoleApp.exe started with an endless loop");
-                        Thread.Sleep(30 * 1000);
-                    }
-                    if (args[0].Equals("error"))
-                    {
-                        Exception ex = new Exception("Intentional Exception thrown");
-                        throw ex;
-                    }
-                    if (args[0].Equals("input"))
-                    {
-                        string str = Console.ReadLine();
-                    }
+                    // To avoid potential issues with orphaned processes (say, the test exits before
+                    // exiting the process), we'll say "infinite" is actually 30 seconds.
+                    Thread.Sleep(30 * 1000);
                 }
-                else
+                else if (args[0].Equals("error"))
+                {
+                    Console.Error.WriteLine("ProcessTest_ConsoleApp.exe error stream");
+                    DelayTask();
```

Comment:
What's the purpose of these DelayTask() calls?  Seems like it's sleeping for 100ms, and if so, is that setting us up for lab failures if the timing is important?